SmartTutor/ContentModel/LearningObjects/ChallengeModel/FulfillmentStrategy/ChallengeFulfillmentStrategy.cs

```diff
@@ -16,5 +16,25 @@ protected List<CaDETMember> GetMethodsFromClasses(List<CaDETClass> classes)
         {
             return classes.SelectMany(c => c.Members.Where(m => m.Type.Equals(CaDETMemberType.Method))).ToList();
         }
+
+        protected List<CaDETField> GetFieldsFromClasses(List<CaDETClass> classes)
```

Comment:
This is not logic required by the MetricChecker, so we should move it to the child that requires it to avoid [Refused Bequest](https://refactoring.guru/smells/refused-bequest).

Reply:
Currently, there is not one regular method in this class which is called by multiple children, so I moved method GetMethodsFromClasses too (other child uses get members).